third_party/move/move-vm/runtime/src/loader.rs

```diff
@@ -650,8 +650,7 @@ impl Loader {
         ) {
```

Comment:
Could it be better to use map_err here? I see we do early return in err case, but maybe that can be restructured as well, e.g. by chained .map(|script| self.verify_script(&script)) (and some unwrap_else magic maybe) or just separate return. Just pointing out, whatever you think makes more sense.